### DISCUSSION TITLE
Speech enhancement tasks

### DISCUSSION THREAD

**Discussion by tgthuan** [yoshitomo-matsubara/torchdistill#321] [Q&A] [answered]
I have recently discovered this toolkit and I find it very interesting. I would like to ask if it is possible to use this toolkit for speech enhancement tasks.
@yoshitomo-matsubara

**Comment by yoshitomo-matsubara** [comment, marked as answer]
Hi @tgthuan 

It should be possible though I am not familiar with the domain.

**Reply by tgthuan** [reply]
I really appreciate it!
